app/components/DetailsDropdownButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button, Icon } from '@trussworks/react-uswds'
+import type { ButtonProps } from '@trussworks/react-uswds/lib/components/Button/Button'
+import type { ReactNode } from 'react'
+
+export default function DetailsDropdownButton({
+  children,
+  className,
+  ...props
+}: {
+  children: ReactNode
+  className?: string
+} & Omit<ButtonProps & JSX.IntrinsicElements['button'], 'type'>) {
+  return (
+    <Button
+      className={`usa-button ${className ?? ''}`}
+      type="button"
+      aria-controls="details-content"
+      role="button"
```

Comment:
Isn't this role implied by `type="button"`?

Reply:
Apparently so. I have not used the ARIA stuff that much, didn't realize it did that